src/eggd_CANRISK_vcf.sh

```
@@ -84,7 +84,7 @@ main() {
 
     mark-section "Checking for low coverage"
     # identify variants with low coverage (below depth threshold input)
-    filter="FORMAT/DP<$depth"
+    filter="FORMAT/DP<$depth && CHROM!="X""
```

Shellcheck (0.10.0)

[warning] depth is referenced but not assigned.

(SC2154)